From f10104ecf6f6135da7043a0c6a2f6c207ea0a92c Mon Sep 17 00:00:00 2001
From: Martin Kepplinger <martink@posteo.de>
Date: Fri, 1 Sep 2017 07:22:09 +0200
Subject: [PATCH] configure.ac: add GNU/kFreeBSD to build config

In order to build the same programs like for FreeBSD

Signed-off-by: Martin Kepplinger <martink@posteo.de>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index fce856d..46f2b7f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -28,7 +28,7 @@ case "${host_os}" in
 	linux*)
 		build_linux=yes
 		;;
-	freebsd*)
+	freebsd*|k*bsd-gnu*)
 		build_freebsd=yes
 		;;
 	gnu*)
-- 
2.11.0

